wpimath/src/main/native/include/frc/kinematics/SwerveDriveKinematics.h

```diff
@@ -42,6 +42,9 @@ namespace frc {
  *
  * Forward kinematics is also used for odometry -- determining the position of
  * the robot on the field using encoders and a gyro.
+ * @see <a
```

Comment:
Put a newline between the previous paragraph and the `@see` directive to match Java.